USAGE.md

```diff
@@ -53,28 +53,21 @@ then you have to create a directory called `msgpack` at the root of your reposit
 
 ## About the CI setup
 
-You'll still need to setup TravisCI to take advantage of the included `.travis.yml` file.  To do this, you'll need:
+You'll still need to setup CircleCI to take advantage of the included `.travis.yml` file.  To do this, you'll need:
```

Comment:
This still says `.travis.yml`, which is not the right config file for Circle CI.